src/testsets.jl

```diff
@@ -186,12 +229,23 @@ function handle_top_level_results!(ts::AbstractTestSet)
         ts.results = AbstractTestSet[]
         ts_nested = ReportingTestSet("Top level tests")
         ts_nested.results = original_results[isa_Result]
+        ts_nested.time_taken = sum(map(x -> x.time_taken, ts_nested.results))
```

Comment:
I think this can just be


```suggestion
        ts_nested.time_taken = sum(x -> x.time_taken, ts_nested.results)
```

Reply:
Oh nice, didn't realise you could do this with `sum`